### DISCUSSION TITLE
Don't replace mouse buttons in osu!standart.

### DISCUSSION THREAD

**Discussion by sasha17102001** [ppy/osu#26620] [Ideas] [answered]
Can you add an option to change the 3 and 4 buttons for osu!standard mode in the settings, so that they don’t replace the mouse buttons but act as additional ones? I want to play with K1K2K3K4M1M2. MCosu kind of has this feature, but it doesn’t show K3 and K4. Of course make it impossible to press simultenously buttons like K1K3M1.

**Comment by peppy** [comment, marked as answer]
this is not going to happen

**Comment by Gabixel** [comment]
To be clear, you're proposing the possibility of using more than 4 keys at a time?
Searching a little bit I found #20423. Is it by any chance about/similar to your proposal?

**Reply by sasha17102001** [reply]
It's kind of similar, but i'm talking only about allowing to use mouse buttons when you have m1 and m2 replaced in settings. So you can use 4 keys on keyboard and 2 mouse buttons. Making it 6 keys(buttons).

**Reply by Gabixel** [reply]
I see. But yeah, seems not to be something that will happen, from peppy's answer. I'm sorry
